modules/client/package.json

```diff
@@ -61,7 +61,7 @@
   },
   "dependencies": {
     "@aragon/core-contracts-ethers": "^0.7.1-alpha",
-    "@aragon/sdk-common": "^0.10.0-beta",
+    "@aragon/sdk-common": "^0.11.0-beta",
     "@aragon/sdk-ipfs": "^0.2.0-alpha",
```

Comment:
Could we also force a bump on sdk-ipfs to `0.3.0-beta` as well?

Reply:
done